apps/docs-snippets/src/guide/wallets/connectors.test.ts

```diff
@@ -0,0 +1,221 @@
+    // #endregion fuel-connector-events-connection
+
+    // #region fuel-connector-events-connectors
+    const connectors: Array<FuelConnector> = [new MyWalletConnector()];
+
+    this.emit(this.events.connectors, connectors);
+    // #endregion fuel-connector-events-connectors
+
+    // #region fuel-connector-events-currentConnector
+    const currentConnector: FuelConnector = new MyWalletConnector();
+
+    this.emit(this.events.currentConnector, currentConnector);
+    // #endregion fuel-connector-events-currentConnector
+
+    // #region fuel-connector-events-currentAccount
+    const currentAccount: string = '0x1234567890abcdef';
+
+    this.emit(this.events.currentAccount, currentAccount);
+    // #endregion fuel-connector-events-currentAccount
+
+    // #region fuel-connector-events-networks
+    // #import { Network };
+
+    const network: Network = {
+      chainId: 1,
+      url: 'https://example.com/rpc',
+    };
+
+    this.emit(this.events.networks, network);
+    // #endregion fuel-connector-events-networks
+
+    // #region fuel-connector-events-currentNetwork
+    // #import { Network };
+
+    const currentNetwork: Network = {
+      chainId: 1,
+      url: 'https://example.com/rpc',
+    };
+
+    this.emit(this.events.currentNetwork, currentNetwork);
+    // #endregion fuel-connector-events-currentNetwork
+
+    // #region fuel-connector-events-assets
+    // #import { Asset };
+
+    const assets: Array<Asset> = [
+      {
+        name: 'Ethereum',
+        symbol: 'ETH',
+        icon: 'https://cdn.fuel.network/assets/eth.svg',
+        networks: [
+          {
+            type: 'ethereum',
+            chainId: 11155111,
+            decimals: 18,
+          },
+        ],
+      },
+    ];
+
+    this.emit(this.events.assets, assets);
+    // #endregion fuel-connector-events-assets
+
+    // #region fuel-connector-events-abis
+    const abis: Array<FuelABI> = [
+      {
+        encoding: '1',
+        types: [],
+        loggedTypes: [],
+        functions: [],
+        messagesTypes: [],
+        configurables: [],
+      },
+    ];
+
+    this.emit(this.events.abis, abis);
+    // #endregion fuel-connector-events-abis
+  }
+}
+
+/**
+ * @group node
+ */
+describe('connectors', () => {
+  describe('instantiation', () => {
+    test('should be able to instantiate a Fuel SDK', () => {
+      // #region fuel-instantiation-1
+      // #import { Fuel };
+
+      const sdk = new Fuel();
+      // #endregion fuel-instantiation-1
+
+      expect(sdk).toBeDefined();
+    });
+
+    test('should be able to instantiate with connectors', () => {
+      const defaultConnectors = (_opts: { devMode: boolean }): Array<FuelConnector> => [
+        new WalletConnector(),
+      ];
+
+      // #region fuel-options-connectors
+      // #import { Fuel };
+      // #context import { defaultConnectors } from '@fuels/connectors';
+
+      const sdk = new Fuel({
+        connectors: defaultConnectors({
+          devMode: true,
```

Comment:
@petertonysmith94 do you think it is worth talking about the `devMode` flag?

Reply:
26b3e01aa15feadec77d3f23267cfff9e18544eb